Piwik.Tracker/PiwikTracker.cs

```diff
@@ -431,8 +429,7 @@ public void ClearCustomTrackingParameters()
         /// </summary>
         public void SetNewVisitorId()
         {
-            var encodedGuidBytes = new MD5CryptoServiceProvider().ComputeHash(Encoding.Default.GetBytes(Guid.NewGuid().ToString()));
-            _randomVisitorId = BitConverter.ToString(encodedGuidBytes).Replace("-", "").Substring(0, LengthVisitorId).ToLower();
+            _randomVisitorId = Guid.NewGuid().ToByteArray().ToSha1().Substring(0, LengthVisitorId).ToLower();

```

Comment:
Not a big deal but I think `ToLower` is not required here since `ToSha1` produces lower case chars right?